torchao/dtypes/uintx/uintx.py

```diff
@@ -55,6 +56,7 @@ class UintxTensor(TorchAOBaseTensor):
     def __new__(
         cls,
         shards: List[torch.Tensor],
+        original_size: int,
```

Comment:
what is this parameter used for? Just seems like its getting passed around

Reply:
It is used in the unpack function 
def get_plain(self):
        return unpack(self.get_shards(), original_size=self.original_size, elem_size=self.bit_width, dim = self.pack_dim)